apps/search/js/providers/suggestions.js

```diff
@@ -59,6 +69,8 @@
     },
 
     search: function(input, preventRemote) {
+
+      this.currentSearch = input;
```

Comment:
I think the only way to handle this is to keep a reference to the currentSearch somewhere, however I did miss clearing the input, good catch